include/InlineEditing/InlineEditing.php

```diff
@@ -128,6 +128,7 @@ function getEditFieldHTML($module, $fieldname, $aow_field, $view = 'EditView', $
     $validMultipleElementNames = array(
```

Comment:
An alternative approach to doing a type check for the input field you're turning into an inline edit that I thought would be adding a attribute to the field in the template to detect if it should be handled as an array or not. The problem with this is function inputs such as email currently fall over when they're treated as a multiple editable section.

Primary is used in the detail view while "billing_..." is used elsewhere, where there is inline Editing. Without some approach to know which fields should become inline editable, with the current layout structure, fields such as "Country" under default sections like "Contacts" turn into multi-edit field as they retrieve across multiple columns, even if they're displaying the one column to the user.

Currently looking into the attribute approach instead as I'd prefer to apply an attribute to trigger multiple elements rather than appending to an array.